playground/.changeset/itchy-bees-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+"playground": minor
+---
+
+fix: location of test db, awaiting strapi teardown
```

Comment:
The playground doesn't need a changeset as it won't be published. This file can be deleted.